translation-config.yml

```diff
@@ -1,6 +1,46 @@
+locales:
+  default: en_US
+  supported:
+    - ar
+    - de
+    - en_GB
```

Comment:
Should `en_US` itself be in the list of supported locales?

Reply:
My reasoning is that by defining it as the default locale, it is implicit.